kibana/templates/service.yaml

```diff
@@ -3,10 +3,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "kibana.fullname" . }}
-  labels:
-    app: {{ .Chart.Name }}
-    release: {{ .Release.Name | quote }}
-    heritage: {{ .Release.Service }}
+  labels: {{ include "kibana.labels" . | nindent 4 }}
```

Comment:
This change has been implemented in https://github.com/elastic/helm-charts/pull/555. Can you merge master in your branch?

Reply:
Merged